packages/tools/kadena-cli/src/account/accountOptions.ts

```diff
@@ -224,4 +225,13 @@ export const accountOptions = {
       'Public keys to add to account',
     ),
   }),
+  confirmAccountVerification: createOption({
+    key: 'verify' as const,
+    validation: z.string(),
+    prompt: account.confirmAccountVerificationPrompt,
+    option: new Option(
+      '-v, --verify <verify>',
```

Comment:
is this supposed to be a boolean flag? or is there an actual value you're expecting to get. I think it should be `'-v, --verify'`

Reply:
I'm expecting the value `true` or `false` explicitly from the user

Reply:
That's not consistent then though, i've used the boolean-flag on other commands where you would just have to pass `--verify`.

Reply:
okay! makes sense! will update it